src/rider/main/kotlin/com/jetbrains/rider/plugins/efcore/features/migrations/add/AddMigrationDialogWrapper.kt

```diff
@@ -101,25 +100,25 @@ class AddMigrationDialogWrapper(
             }
             row {
                 checkBox(EfCoreUiBundle.message("checkbox.open.migration.file.after.executing"))
-                    .bindSelected(dataCtx.openMigrationFile)
+                    .bindSelected(dataCtx.openMigrationFileAfterExecuting)
             }
         }
     }
 
     override fun preCommandExecute() {
-        if (!dataCtx.openMigrationFile.value)
+        if (!dataCtx.openMigrationFileAfterExecuting.value)
             return
 
         val migrationsOutputFolderPath = Path(migrationProjectFolder.value)
             .resolve(dataCtx.migrationsOutputFolder.value).pathString
 
-        intellijProject.service<OpenMigrationFileService>()
+        OpenMigrationFileService.getInstance(intellijProject)
```

Comment:
Since we are using this service a couple of times, lets extract it to a private field:
```kotlin
private val openMigrationFileService by lazy { OpenMigrationFileService.getInstance(intellijProject) }
```